tsconfig.json

```diff
@@ -1,5 +1,7 @@
 {
   "compilerOptions": {
+    "importHelpers": true,
```

Comment:
Noted, thanks... I'm going to attempt to update TS... 

Sadly, that makes this a breaking change. :(